hack/quickstart/openstack/config.tf

```diff
@@ -0,0 +1,24 @@
+variable "key_pair" {
+  type    = "string"
+  default = "philips"
+}
+
+variable "flavor_id" {
+  type    = "string"
+  default = "bbcb7eb5-5c8d-498f-9d7e-307c575d3566"
```

Comment:
This seems to be provider specific, openstack comes with defaults, but vexxhost has its own. I'll add a small chapter on how to list flavors.